values.yaml

```diff
@@ -65,6 +65,8 @@ global:
 
   # persistent-volume and associated claims configurations
   persistence:
+    storageClassDeploy: false
```

Comment:
I don't think you need this variable

Reply:
The storage class changes are normally done at the point of EKS pipeline creation at the point of the efs csi driver instantiation. I wanted to include the definition here so that if a deployment was not done correctly, the user was aware of the changes that might be needed.